hermeto/core/models/input.py

```diff
@@ -43,6 +44,22 @@ def show_error(error: "ErrorDict") -> str:
         location = " -> ".join(map(str, error["loc"]))
         message = error["msg"]
 
+        if "expected tags:" in message:
```

Comment:
This looks rather fishy: one should not rely on parsing error messages unless they absolutely must. Where do these tags come from? It is likely better to address this problem there, at the root.

Reply:
Agreed. The error message is already totally user unfriendly (pre-existing).